training/general/src/environments/acpt-pytorch-1.11-py38-cuda11.5-gpu/context/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu115-py38-torch1110:{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}
+
+# Install pip dependencies
+RUN pip install 'ipykernel~=6.0' \
```

Comment:
Please consider moving these and the packages below to a requirements.txt file and install with one call to `pip install`.